dataminer-overview/DataMiner_Stack_Functions/Stack_Data_Visualization.md

```diff
@@ -58,48 +58,168 @@ In systems with a Cassandra and Elasticsearch database, DataMiner’s [Augmented
+
+With the current trends in the industry, this challenge is growing even more daunting by the day. The main cause of inadequate fault management is a loosely coupled architecture and the attempt to aggregate faults generated by disparate systems. **By tapping into every piece of the actual installed base, DataMiner provides a real-time, end-to-end view of the complete ecosystem**. As one of the most powerful fault management solutions in the industry, DataMiner offers a **suite of fault management-related innovations** that enable operators to pinpoint and resolve operational issues, faster than ever before by providing them with an **intuitive multi-user management of active and historical alarms**.
+
+These innovations range from small gems such as highly user-definable device icons, which offer historical fault context information at a glance, to a notification banner with service impact analysis data, and revamped data tables with heatmap overlays and integrated histogram analysis. Furthermore, the core engine offers [**sophisticated off-the-shelf, self-learning algorithms**](xref:Stack_Augmented_Operations) to intelligently and effortlessly monitor and track the most challenging operational metrics, including traffic loads, slowly degrading quality metrics, and much more.
+
+Today, **raw data collection** is the foundation of an efficient architecture. It maximizes the availability of information for the management platform to carry out its functions.
+
+To address the challenges of fault detection, ensuring **good data hygiene** is key. DataMiner includes various ways to guarantee data hygiene, including:
+
+- Fully user-definable and run-time-applied alarm thresholds defined in templates.
+- Dynamic switching of alarm definitions based on time or events (e.g. upon service activation).
+- Ability to stack alarm templates to easily and efficiently manage thresholds across vast deployments.
+- Support for relative and absolute thresholds, including a variety of baseline methods.
+- Hysteresis on trigger and/or clear, applicable to all individual thresholds.
+- Conditional alarming to prevent unnecessary alarm clutter.
+
+**Unified fault detection** is one of the most fundamental and critical functionalities of a management platform. DataMiner offers a unified approach with significant advantages:
 
-## Fault Management
+- Gaining massive amounts of time to define and apply thresholds across your entire operation.
+- Customizing alarm definitions exactly the way your operation requires, eliminating noisy alarms.
+- Triggering alarms in a far more intelligent way and reducing the alarm count to what is truly necessary.
+- Conserving valuable resources and time by reducing the need to resolve shortcomings with complex and cumbersome event correlation.
+- Significantly reducing Mean Time to Repair (MTTR), increasing optional efficiency, lowering operational costs, and increasing Quality of Service (QoS).
```

Comment:
```suggestion
- Significantly reducing Mean Time to Repair (MTTR), increasing operational efficiency, lowering operational costs, and increasing Quality of Service (QoS).
```
Is this supposed to be "operational" instead of "optional", @SammyDW ?

Reply:
You are right, Edith, thank you for noticing, common typo it seems.